### PR TITLE
Bump versions; migrate to androidx

### PR DESCRIPTION
**Pull request by markormesher**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ buildscript {
 	}
 
 	dependencies {
-		classpath 'com.android.tools.build:gradle:3.1.3'
+		classpath 'com.android.tools.build:gradle:3.2.1'
```
Comment (Nimrodda) [inline app/build.gradle:11]: there's a newer version 3.3.1

File: fab/build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
 	}
 
 	dependencies {
-		classpath 'com.android.tools.build:gradle:3.1.2'
+		classpath 'com.android.tools.build:gradle:3.2.1'
```
Comment (Nimrodda) [inline fab/build.gradle:11]: there's a newer version 3.3.1

File: app/build.gradle (hunk 1 above)
Comment (markormesher) [reply app/build.gradle:11]: There is, but it has compatibility issues with generated code, such as the `R` lookup file and some Kotlin-generated sources.

**Comment by markormesher** [comment]
Testing well on my device. Waiting to test on an older device before merging and cutting a release.

**Pull request merged**